### ISSUE TITLE
plugin: unknown plugin type

### ISSUE THREAD

**Issue by poloyacero** [traefik/plugindemo#17]
I'ved created and installed a simple plugin, https://plugins.traefik.io/plugins/633ea0230362b66628c949d6/header-authorization

Have it installed in my traefik instance, using helm

**traefik-values.yaml**

experimental:
  http3:
    enabled: false
  plugins:
    enabled: true
    headauth:
      moduleName: "github.com/poloyacero/headauth"
      version: "v0.0.1"
      
**dynamic_configuration**

apiVersion: traefik.containo.us/v1alpha1
kind: Middleware
metadata:
  name: headerauth
spec:
  plugin:
    headauth:
      header:
        name: "X-User-Role"
      allowed:
        - "super_admin"
      methods:
        - PATCH
        - DELETE
        - POST
        
Basically just went through the instructions from Plugin Catalog but when I use the middleware plugin and have it assign to a route I got this error,
![image](https://user-images.githubusercontent.com/9990246/194338400-fe9262a8-d3ca-4162-bccf-067aabad0d59.png)

Please let me know if I missed something. Thank you so much...
